File: ProveHimWrong.cpp
#include <bits/stdc++.h>

typedef long long ll;

using namespace std;


int main(){
    int t;
    cin >> t;
    while (t--){
        ll n;
        cin >> n;
        if (n > 19){
            cout << "NO";
        } else{
            cout << "YES" << "\n";
            for (ll i=0; i < n; i++){
                cout << (ll)pow(3, i) << (i == n-1 ? "" : " ");
            }
        }
        cout << "\n";
    }
}

File: ProveHimWrong.py
from math import pow
for _ in range(int(input())):
    n = int(input())
    if n > 19:
        print("NO")
    else:
        print("YES")
        for  i in range(n):
            if i == n-1:
                print(pow(3, i))
            else:
                print(pow(3, i), end=" ", sep=" ")
